### ISSUE TITLE
[Question] output topics representation

### ISSUE THREAD

**Issue by yuyingg0921** [hugochan/KATE#14]
Hello!
![image](https://user-images.githubusercontent.com/33712951/45466839-f45f0480-b74e-11e8-910d-4f1e4578e06a.png)

This is a part of  my output topic representation, but it looks a little strange because the words seems not so important (the.is.with.will.or.s.some......).
My input training data is 20newsgroup and I separate it into 70% training set. 30% testing set.

I think maybe I did something wrong somewhere.
Do you have any suggestion!?

thank you very much.

**Comment by yuyingg0921** [comment]
Thanks for replying.
but where(what) is the argument?

**Comment by saria85** [comment]
well, logically it should be where you are building your corpus. probably in pre-processing phase, then search word "stop" you will see that in one or two methods as an argument, I think there was stem= False so if you need to apply stemming you also have to set it True.

**Comment by saria85** [comment]
you are welcome.
He helped me during my exploring the code so I feel like to compensate :)
Actually I did not try Accuracy as I had unlabeled data. but from my experience on a different data set you should get good result. That would be good to check the source code maybe you should set a parameter, .. something.

**Comment by hugochan** [comment]
@yuyingg0921 Are you using the run_classifier.py script for the classification task? If so, what are your parameters? You can also visualize the document vectors to get a sense of the quality.

**Comment by yuyingg0921** [comment]
@hugochan  oh sorry I didn't see the run_classifier.py file so I built one.
It seems great now, I got accuracy about 60-70%.
Thank you all very much! You helped me a lot.
